ui/partials/perf/comparectrl.html

```diff
@@ -10,8 +10,8 @@
     <div id="subtest-summary" ng-if="!dataLoading && !errors.length">
       <h1>Perfherder Compare Revisions</h1>
       <ul class="list-inline push-information">
-        <li><strong>Base</strong> - <a href="{{originalProject.getRevisionHref(originalRevision)}}">{{originalRevision}}</a> ({{originalProject.name}})</li>
-        <li><strong>New</strong> - <a href="{{newProject.getRevisionHref(newRevision)}}">{{newRevision}}</a> ({{newProject.name}})</li>
+        <li><strong>Base</strong> - <a href="{{originalRevision|getRevisionUrl:originalProject.name}}">{{originalRevision}}</a> ({{originalProject.name}})</li>
+        <li><strong>New</strong> - <a href="{{newRevision|getRevisionUrl:newProject.name}}">{{newRevision}}</a> ({{newProject.name}})</li>
```

Comment:
It looks like we're going to handle the directive in bug 1181573, so ignore this comment.